HW0/P6/P6B.py

```diff
@@ -10,22 +11,37 @@ def burnTime(t):
 if __name__ == '__main__':
     N = 16  # The number of jobs
     P = 4   # The number of processes
+    Tests = 100
 
     # A thread pool of P processes
     pool = mp.Pool(P)
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = np.logspace(-6, 0, Tests)
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+
+        start = time.time()
+        for x in range(0,16):
+            burnTime(wait_time[t])
```

Comment:
I think variable t is one element in wait_time. And I think this will raise error if t is not an integer. If you want to iterate through both index and element, this would be a better way:
for index, time in enumerate(wait_time)